RecoJets/FFTJetProducers/src/FFTJetParameterParser.cc

```diff
@@ -387,23 +389,23 @@ namespace fftjetcms {
     return return_type(nullptr);
   }
 
-  std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long> > fftjet_ClusteringTreeSparsifier_parser(
+  std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>> fftjet_ClusteringTreeSparsifier_parser(
       const edm::ParameterSet& ps) {
-    typedef std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long> > return_type;
+    typedef std::unique_ptr<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>> return_type;
 
     const int maxLevelNumber = ps.getParameter<int>("maxLevelNumber");
     const unsigned filterMask = ps.getParameter<unsigned>("filterMask");
-    const std::vector<double> userScalesV = ps.getParameter<std::vector<double> >("userScales");
+    const std::vector<double> userScalesV = ps.getParameter<std::vector<double>>("userScales");
     const unsigned nUserScales = userScalesV.size();
     const double* userScales = nUserScales ? &userScalesV[0] : nullptr;
 
-    return return_type(
-        new fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>(maxLevelNumber, filterMask, userScales, nUserScales));
+    return std::make_unique<fftjet::ClusteringTreeSparsifier<fftjet::Peak, long>>(
```

Comment:
Why clang is not also fixing the similar line 414 below?

Reply:
I'll guess that in that case the check wasn't smart enough to substitute the body of the return_type variable, there are other cases like this one on lines 294 and 313